src/isomorphic/modern/class/__tests__/ReactClassEquivalence-test.js

```diff
@@ -32,7 +32,7 @@ describe('ReactClassEquivalence', function() {
 function runJest(testFile) {
   var cwd = process.cwd();
   var jestBin = path.resolve('node_modules', '.bin', 'jest');
-  var setupFile = path.resolve(__dirname, 'setupSpecEquivalenceReporter.js');
+  var setupFile = path.resolve(__dirname, 'setupSpecEquivalenceReporter.js.setup');
```

Comment:
What about putting it where we have other jasmine-related setups: root/scripts/jest? We're already making the assumption that `path.resolve` is going to work from our repo root so it just means we change `__dirname` to `scripts/jest`).